classes/fields/avatar.php

```diff
@@ -144,7 +145,7 @@ public function get_avatar( $avatar, $id_or_email, $size, $default = '', $alt =
 							$attributes['alt'] = $alt;
 						}
 
-						$user_avatar = pods_image( $user_avatar, array( $size, $size ), 0, $attributes );
+						$user_avatar = pods_image( $user_avatar, array( $size, $size ), 0, $attributes, true );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 1226922a44a6b7258ec786ffa346cadc</sub>